src/main/java/com/redhat/darcy/webdriver/internal/WebDriverTarget.java

```diff
@@ -31,5 +31,9 @@
  * @see WebDriverTargets
  */
 public interface WebDriverTarget {
+    /**
+     * @throws org.openqa.selenium.NotFoundException if this target is not able to be switched to,
```

Comment:
Yeah typically subclasses of `NotFoundException` are thrown such as `NoSuchWindowException`. This is the general behavior of WebDriver's `TargetLocator`s.

What's interesting here is that the `TargetedTargetLocator` does _not_ work this way -- it will never throw an exception, even if the target is not found. This meant that using a target that did -- such as the new "view" target -- made that target behave differently from other targets. So, I added that as an explicit contract of WebDriver targets. As I said, currently targets that are doing their switching using a `TargetedTargetLocator` do not actually behave this way. This is an odd inconsistency and has made me question if that target locator should exist at all. Something to think about.